Dockerfile

```diff
@@ -45,19 +47,9 @@ ARG GIT_BRANCH="develop"
 RUN ~/src/carma-platform/docker/checkout.bash -b ${GIT_BRANCH}
 
 COPY --chown=carma . /home/carma/src/carma-platform
-# /////////////////////////////////////////////////////////////////////////////
-# Stage 2 - Build and install the software
-# /////////////////////////////////////////////////////////////////////////////
-
-FROM base-image AS install
-ARG PACKAGES=""
-ENV PACKAGES=${PACKAGES}
-
-# Copy the source files from the previous stage and build/install
-RUN mkdir ~/carma_ws
-COPY --from=source-code --chown=carma /home/carma/src /home/carma/carma_ws/src
 
-RUN ~/carma_ws/src/carma-platform/docker/install.sh
+## Build and install the software
+RUN ~/src/carma-platform/docker/install.sh
 
 # /////////////////////////////////////////////////////////////////////////////
 # Stage 3 - Finalize deployment
```

Comment:
Stage 2 is removed it looks like. Can we still keep that `install` stage?

Reply:
Based on in person conversation, keeping the current changes with the checkout and install called on same step - since this avoids having to copy the source code twice.